tests/test_collect_test_data_results.py

```diff
@@ -43,24 +43,26 @@ def local_test_spectra():
 
 
 def test_generate_test_results_ms2query(ms2library, local_test_spectra):
-    result = generate_test_results_ms2query(ms2library, local_test_spectra)
-    np.testing.assert_almost_equal(result[0], (0.5645, 0.003861003861003861, False))
-    np.testing.assert_almost_equal(result[1], (0.409, 0.010610079575596816, False))
+    results = generate_test_results_ms2query(ms2library, local_test_spectra)
```

Comment:
@florian-huber We had many tests that check for specific outcomes of the models. This is not easily maintainable, here I have removed all these specific checks and replace with checks for correct types instead of values. Do you think this is the way to go? Or do you have any other suggestions?